aspnetcore/fundamentals/logging/loggermessage/sample/Internal/LoggerExtensions.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace LoggerMessageSample.Internal
+{
+    public static class LoggerExtensions
+    {
+        #region snippet1
+        private static readonly Action<ILogger, Exception> _indexPageRequested;
+        #endregion
+
+        #region snippet2
+        private static readonly Action<ILogger, string, Exception> _quoteAdded;
+        #endregion
+
+        #region snippet3
+        private static readonly Action<ILogger, string, int, Exception> _quoteDeleted;
+        private static readonly Action<ILogger, int, Exception> _quoteDeleteFailed;
+        #endregion
+
+        #region snippet4
+        private static Func<ILogger, int, IDisposable> _allQuotesDeletedScope;
+        #endregion
+
+        static LoggerExtensions()
+        {
+            #region snippet5
+            _indexPageRequested = LoggerMessage.Define(
+                LogLevel.Information, 
+                1, 
```

Comment:
Yeah, throw a name in there, it makes things clearer in structured logging stores :)

Reply:
Done :+1:

Reply:
> makes things clearer in structured logging stores 

ah ... that must be the *other reason* besides just being a good visual aid. I'll add that on the next commit. Thx.

Reply:
@anurse ... but with regard to the placeholder names being descriptive ... do structured logging stores also use those names?

Reply:
The name is made available to logging providers via `EventId.Name`, it's up to the logging store to do something with it. But we want to strongly encourage users to include names since it's easier to update providers to use the names than to update logging sites to provide the names :)

Reply:
But yeah, it does indeed look like Serilog uses it (as an example): https://github.com/serilog/serilog-extensions-logging/blob/dev/src/Serilog.Extensions.Logging/Extensions/Logging/SerilogLogger.cs#L157